internal/v2/resources/apono_resource_integration_resource.go

```diff
@@ -247,6 +265,14 @@ func (r *AponoResourceIntegrationResource) Read(ctx context.Context, req resourc
 		return
 	}
 
+	if integration.Category != common.ResourceCategory {
```

Comment:
this will cause import to fail if it's not a resource.
(also added a unit test for it).